go/interpreter/lfvm/instructions.go

```diff
@@ -919,6 +919,8 @@ func genericCreate(c *context, kind tosca.CallKind) error {
 	gas := c.gas
 	gas -= gas / 64
 	if err := c.useGas(gas); err != nil {
+		// this usage can never fail because the endowment is at most
+		// 63/64 of the current gas level.
```

Comment:
actually I prefer having the coverage exception (and unreachable code) before not checking for an error. In case anything changes in the future, if the error is ignored, we would never know if something breaks. 

@HerbertJordan what do you think ?

Reply:
If you two have already discussed this, please ignore my comment and carry on.

Reply:
Idea: merge this part with the line 942, where the left-over gas of the call is returned (and where encapsulation is broken right now).

Instead subtracting all the gas given to the recursive call initially, and then re-adding whatever is left, just charge the consumed gas once, after the call.